autopush/utils.py

```diff
@@ -375,9 +375,11 @@ def cleanup_headers(self):
         headers = self.headers
         # Strip crypto/encryption headers down
         for hdr in ["crypto-key", "encryption"]:
-            if STRIP_PADDING.search(headers.get(hdr, "")):
+            if hdr in headers:
                 head = headers[hdr].replace('"', '')
-                headers[hdr] = STRIP_PADDING.sub("", head)
+                head = STRIP_PADDING.sub("", head)
+                head.decode('ascii')
```

Comment:
`decode` returns a string (it doesn't mutate), but the output is not used here?

Reply:
It's more a check to see if the header response is ascii and bailing if it's not. While it's very unlikely that anyone were to send a header that is `salt="💩"`, this would catch it. I'm also fine dropping it, but I tend to be paranoid and if we're doing checks to safeguard the client, figured I might as well add it.